tests/cpp/test_host_ir_integration.cpp

```diff
@@ -27,49 +29,55 @@ using HostIrEvaluatorTest = NVFuserTest;
 // using HostIrEvaluator.
 TEST_F(HostIrEvaluatorTest, LaunchKernel) {
   Fusion fusion;
-  FusionGuard fg(&fusion);
-  TensorView* in = makeSymbolicTensor(2);
-  fusion.addInput(in);
-
-  TensorView* out = set(in);
-  fusion.addOutput(out);
+  {
```

Comment:
Separate the region that creates the fusion and the region that creates the HostIrContainer. This way, FusionGuard can be localized.